package.json

```diff
@@ -86,12 +86,13 @@
     "cordova-plugin-whitelist": "^1.3.3",
     "cordova-sqlite-storage": "^3.2.0",
     "cordova-support-android-plugin": "^1.0.1",
-    "core-js": "^3.1.2",
+    "core-js": "^3.1.3",
     "ionic-angular": "^3.9.6",
     "ionic-plugin-keyboard": "^2.2.1",
     "ionicons": "^4.5.8",
     "kafka-rest": "^0.0.5",
     "moment": "^2.22.2",
+    "ng2-fittext": "^1.2.6",
```

Comment:
what happens if the container is scrollable and then we use fittext=true? Will it shrink the size of the text ? Or will it keep it as it is since it can be scrolled ?

Reply:
It would still shrink the text since it will follow the size of the container and ignore scroll.

Reply:
Also a follow up. What happens when Accessibility settings are enabled ?

Reply:
Currently, it is temporarily set to ignore the text zoom accessibility settings so the text zooms do not break the UI.

Reply:
I think currently it's fine as it's only used for minor components. But we should revisit these if we plan to employ fittext in questionnaires, tasks, etc